examples/mujoco/ppo/train_ppo.py

```diff
@@ -0,0 +1,199 @@
+    assert process_seeds.max() < 2 ** 32
+
+    args.outdir = experiments.prepare_output_dir(args, args.outdir)
+
+    def make_env(process_idx, test):
+        env = gym.make(args.env)
+        # Use different random seeds for train and test envs
+        process_seed = int(process_seeds[process_idx])
+        env_seed = 2 ** 32 - 1 - process_seed if test else process_seed
+        env.seed(env_seed)
+        # Cast observations to float32 because our model uses float32
+        env = chainerrl.wrappers.CastObservationToFloat32(env)
+        if args.monitor:
+            env = gym.wrappers.Monitor(env, args.outdir)
+        if args.render:
+            env = chainerrl.wrappers.Render(env)
+        return env
+
+    def make_batch_env(test):
+        return chainerrl.envs.MultiprocessVectorEnv(
+            [functools.partial(make_env, idx, test)
+             for idx, env in enumerate(range(args.num_envs))])
+
+    # Only for getting timesteps, and obs-action spaces
+    sample_env = gym.make(args.env)
+    timestep_limit = sample_env.spec.tags.get(
+        'wrapper_config.TimeLimit.max_episode_steps')
+    obs_space = sample_env.observation_space
+    action_space = sample_env.action_space
+    print('Observation space:', obs_space)
+    print('Action space:', action_space)
+
+    assert isinstance(action_space, gym.spaces.Box)
+
+    # Normalize observations based on their empirical mean and variance
+    obs_normalizer = chainerrl.links.EmpiricalNormalization(
+        obs_space.low.size, clip_threshold=5)
+
+    # Orthogonal weight initialization is used as OpenAI Baselines does
+    winit = chainerrl.initializers.Orthogonal(1.)
+    winit_last = chainerrl.initializers.Orthogonal(1e-2)
+
+    action_size = action_space.low.size
+    policy = chainer.Sequential(
+        L.Linear(None, 64, initialW=winit),
+        F.tanh,
+        L.Linear(None, 64, initialW=winit),
+        F.tanh,
+        L.Linear(None, action_size, initialW=winit_last),
+        chainerrl.policies.GaussianHeadWithStateIndependentCovariance(
+            action_size=action_size,
+            var_type='diagonal',
+            var_func=lambda x: F.exp(2 * x),  # Parameterize log std
+            var_param_init=0,  # log std = 0 => std = 1
+        ),
+    )
+
+    vf = chainer.Sequential(
+        L.Linear(None, 64, initialW=winit),
+        F.tanh,
+        L.Linear(None, 64, initialW=winit),
+        F.tanh,
+        L.Linear(None, 1, initialW=winit),
+    )
+
+    # Combine a policy and a value function into a single model
+    model = chainerrl.links.Branched(policy, vf)
+
+    opt = chainer.optimizers.Adam(3e-4, eps=1e-5)
+    opt.setup(model)
+
+    agent = PPO(
+        model,
+        opt,
+        obs_normalizer=obs_normalizer,
+        gpu=args.gpu,
+        update_interval=args.update_interval,
+        minibatch_size=args.batch_size,
+        epochs=args.epochs,
+        clip_eps_vf=None,
```

Comment:
Baselines' PPO does seem to be clipping:
https://github.com/openai/baselines/blob/master/baselines/ppo2/defaults.py#L11
https://github.com/openai/baselines/blob/master/baselines/ppo2/model.py#L147
https://github.com/openai/baselines/blob/master/baselines/ppo2/model.py#L69

Reply:
No problem here, previous version only used clipping on the policy network
https://github.com/Breakend/baselines/blob/50ffe01d254221db75cdb5c2ba0ab51a6da06b0a/baselines/ppo1/pposgd_simple.py#L115